docs/content/tutorial/introduction.mdx

```diff
@@ -42,7 +43,7 @@ def topstories(topstory_ids):
 
 ### Building a DAG of assets
 
-Defining multiple assets creates a DAG (directed acyclic graph) of what assets depend on other assets. This DAG helps you to:
+A set of assets naturally forms a [DAG (directed acyclic graph)](https://en.wikipedia.org/wiki/Directed_acyclic_graph), where the edges correspond to data dependencies between assets. This DAG helps you to:
```

Comment:
I keep getting stuck on `naturally` here and I'm not sure why. Are there other things that could form a DAG? Could this be `A set of assets forms a DAG` and it's still accurate?

That being said, I'm not gonna block you on what is probably pretty nitpicky 😬

Reply:
I think we can take out "naturally" without losing meaning